MediaGalleryIntegration/Plugin/SaveImageInformation.php

```diff
@@ -84,24 +91,28 @@ public function __construct(
         $this->saveAsset = $saveAsset;
         $this->storage = $storage;
         $this->filesystem = $filesystem;
+        $this->generateRenditions = $generateRenditions;
     }
 
     /**
      * Saves asset to media gallery after save image.
      *
      * @param Uploader $subject
      * @param array $result
+     * @return array
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
+     * @throws \Magento\Framework\Exception\ValidatorException
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function afterSave(Uploader $subject, array $result): array
     {
-        $file = $this->splFileInfoFactory->create($result['path'] . '/' . $result['file']);
+        $file = $this->splFileInfoFactory->create($result['path'] . DIRECTORY_SEPARATOR . $result['file']);
         if (!$this->isApplicable($file->getPathName())) {
             return $result;
         }
         $this->saveAsset->execute([$this->createAssetFromFile->execute($file)]);
-        $this->storage->resizeFile($result['path'] . '/' . $result['file']);
-
+        $this->storage->resizeFile($result['path'] . DIRECTORY_SEPARATOR . $result['file']);
```

Comment:
@konarshankar07 @gabrieldagama I would prefer to use `/` instead of `DIRECTORY_SEPARATOR`.

Do you see any possible issues with this approach?

Reply:
I think @gabrieldagama prefer to use `DIRECTORY_SEPARATOR` because of  Windows and Linux have different directory separator but Magento is not compatible with Windows so I think both the approaches is fine for me

Reply:
I usually go for this constant because I find it easier to read, but '/' is fine as well.

Reply:
I think '/' should be fine for Windows if we are constructing paths. Using the `DIRECTORY_SEPARATOR` constant can result into issues where '\' is used in URL links (considering that Magento filesystem can be switched to https adapter).

So functionally I think using `DIRECTORY_SEPARATOR` is more risky for Windows OS comparing to '/' in most cases.

Reply:
So Do you think we need to change all `DIRECTORY_SEPARATOR` to '/'?

Reply:
It's not necessary for now, just shared my thoughts to bring the attention, I believe it requires a discussion to take a final decision and reflect it in our standards